Objects/cellobject.c

```diff
@@ -140,8 +140,19 @@ cell_get_contents(PyCellObject *op, void *closure)
     return op->ob_ref;
 }
 
+int
+cell_set_contents(PyCellObject *op, PyCellObject *obj)
+{
+    Py_INCREF(obj);
+    PyCellObject *tmp_op = (PyCellObject *)(op->ob_ref);
```

Comment:
I believe the 147 to 149 can be replace by a single invocation of the Py_SETREF macro.  See +856 in Include/object.h.